lang/test/org/partiql/lang/thread/ThreadInterruptedTests.kt

```diff
@@ -1,6 +1,7 @@
 package org.partiql.lang.thread
 
 import com.amazon.ion.system.IonSystemBuilder
+import com.amazon.ionelement.api.ionInt
```

Comment:
Unused imports of `Literal` and `.toIonElement`

Reply:
Will remove them in the next commit.